Buried Treasures/buried-treasures.html

```diff
@@ -0,0 +1,129 @@
+<!-- Character sheet for https://buried-treasures.net -->
+
+<div class="row">
+
+<div class="column">
+
+    <div class="title">MAIN CHARACTERISTICS</div>
+    <br>
+    <div class="line"><label>Name:</label><input type="text" name="attr_character_name" /></div>
+    <div class="line"><label>Species:</label><input type="text" name="attr_species" value="Human" /></div>
+    <div class="line"><label>Appearance:</label><input type="text" name="attr_appearance" /></div>
+    <div class="line"><label>Voice:</label><input type="text" name="attr_voice" /></div>
+    <br>
+
+    <table>
```

Comment:
Greetings @yotsov 

We no longer accept sheets that use '<tables>' for form design. These will need to be replaced with proper elements such as `<div>`, `<h1>/<h2>`..., and other such elements. If you like a the style a table gives you then I'd suggest checking out [grid displays](https://css-tricks.com/snippets/css/complete-guide-grid/).

Thanks,
Cassie

Reply:
Hello @clevett !

Thank you for your note.

I was using table tags all over the place, and in my last commit I removed most of them, except for one, which I thought made sense to remain as a table. Do you want me to remove that last one as well? And to not use table tags at all? Just making sure that I understand correctly.

Reply:
Tables can be used for display tabular data which is a rare occurrence in Character Sheets which at their heart are forms. This is a helpful article to understanding the reasons behind avoiding tables, https://www.lifewire.com/dont-use-tables-for-layout-3468941

Reply:
Sure, not a problem for me, I will remove this table too. Thank you @clevett !